### ISSUE TITLE
Ability to define CSS button styles 

### ISSUE THREAD

**Issue by Otienoh** [pdmfc/nova-action-button#6]
Hey Buddy, neat package. is it possible to a the ability to define the button CSS when defining the button? Thinking of something along the lines of `->styles('primary-outline')` or `->styles('info')`.

_dillingham/nova-button packages_ Has it implemented.

**Comment by skoyah** [comment]
Hi @Otienoh ! Thanks for the feedback! 

I think your point is valid. 
This can be a nice feature to have. The only thing I'm not sure is if it should append the classes or completely override them.

I'll tackle this as soon as I get some free time. Also, PRs are always welcomed! :)

**Comment by Otienoh** [comment]
Yes, that is a good question. I think it will make sense to override the styles as one may want to use a `warning` default style for the button. In such a case I am not sure appending will work well.

**Comment by AbdLateef** [comment]
I have the same issue. Did you already make PR @Otienoh  ?
